source/game.roomhandler.studio.bmx

```diff
@@ -881,9 +881,9 @@ Type RoomHandler_Studio Extends TRoomHandler
 			For local p:TProgrammeLicence = EachIn GetProgrammeLicenceCollection().licences.Values()
 				conceptTitles.addLast(p.getTitle().toLower())
 			Next
-			If conceptTitles.contains(pc.getTitle().toLower())
+			If conceptTitles.contains(pc.getTitle().toLower()) Or Not pc.getTitle().toLower().Contains("#")
```

Comment:
`Or Not pc.getTitle().toLower().Contains("#")`
bitte zu
`Or Not pc.getTitle().Contains("#")`
machen ... erspart einen neuen String, das `#` gibt es auch bei Gross- und Kleinschreibung ;-)
(Ich weiss, ist einfach aus Copy-Paste entstanden...)